### PR TITLE
## Milestone 2: 

### PR DESCRIPTION
**Pull request by sneeu-leeu**
### On this branch I:
- Developed the UI of the calculator.
-  Used class-based components.
- Imported calculator and used it in your main component (App.js).
- Tested it in the browser.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/calculator';
 
 function App() {
   return (
     <div className="App">
```
Comment (henatan99) [inline src/App.js]: - [x] This milestone is meant to practice React class components. Kindly refactor the `App` functional component into a class component. :pray:

File: src/App.css [hunk 2]
```diff
@@ -1,39 +1,30 @@
...
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.cal-val {
+  width: 98%;
+  height: 2.5rem;
+  background-color: rgb(187, 187, 187);
 }
```
Comment (henatan99) [inline src/App.css]: - [x] You can have text-align property here to align the input text to the right like the original design. This is a business requirement and you need to stick to it. 
- [x] You can also set the color property here to make the input text just like the original design.
- [x] As you can compare your design with the original design, the division operator needs to be changed. 
image
image

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @sneeu-leeu , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
- [x] All the boilerplate that got created by the CRA (texts, images, and styles) ought to have gotten deleted. I can see you still have some in your public directory. Kindly remove them. :pray: 
- [ ] I think you should update your readme. In fact, there is a default readme generated when you initialize your react app. But, your should update it with a description of your project. The Microverse [readme template](https://github.com/microverseinc/readme-template) is a pretty well-organized template you can make use of. 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sneeu-leeu** [comment]
@henatan99 Thanks for the review! 
Just some notes for the next reviewer:
- The only CSS file left from the boilerplate is index.css and is in use! The rest are used by me. If it really seems like I am missing one, please be more precise with which one you are talking about.

Thanks for the effort to review this PR!

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashbaord. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**